whichone/static/js/main.js

```diff
@@ -23,6 +23,7 @@ var cheaterMode = false;
 var countdownNumberEl = null;
 var countdown = 10;
 var refreshIntervalId = null;
+var ps = 0;
```

Comment:
What is this var for? Might want to consider a better name.

I see its not used in this file.

Reply:
I tried to make it obscure on purpose so it is harder to cheat without getting caught. It stands for previous score and keeps track of what their score should be. This means that if they update their score in the console (userCurrentScore = 10000; for example), it will see that the score has been modified and it enables cheater mode (something we added in earlier versions to detect if lives > 3). When cheater mode is enabled, their high score is not saved, the spotify icons turn red, and there is a custom message at the end of the game

Reply:
Interesting way to prevent cheating 😄 

I would say its preferable to keep code clean and readable and use an obfuscation tool to build the prod version. Also can generate a more compact JS bundle.

I guess this is another reason to avoid globals & prefer scoped variables. Although of course while the comparisons are made on the UI you'll not be able to stop people from cheating I suppose.